CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+## [16.0.5] - ???
```

Comment:
Will replace the date on this before merging.

Reply:
Curious - why add a date? We don't set this anymore elsewhere.

Reply:
Just for consistency with this file. I can leave it off, though.